build/osx/OSConsX.py

```diff
@@ -504,9 +505,9 @@ def emit_app(target, source, env):
     # idea: hide these in the env[]?
     bundle = Dir(str(bundle))  # coerce the bundle target into being a Dir
     contents = Dir(os.path.join(str(bundle), "Contents"))
-    frameworks = Dir(
-        os.path.join(str(contents), "Frameworks")
-    )  # we put both frameworks and standard unix sharedlibs in here
+    # frameworks = Dir(
```

Comment:
There should be a note why this code has been commented out and not deleted

Reply:
The variable is assigned but not used anywhere. To be honest I didn't even try to understand the code and just fixed code style and python2-only constructs, so I thought I better keep it just in case... We'll remove this file anyway as soon as 2.3 has been branched off (at least I hope we do)

Reply:
But with the code now commented out this information would be lost. So leave a comment why you commented it out, i.e. previously unused variable and no idea why it has been left in the code.